apps/mosaico/src/Mosaico/Frontpage/Models.purs

```diff
@@ -0,0 +1,46 @@
+module Mosaico.Frontpage.Models where
+
+import Prelude
+
+import Data.Array (head)
+import Data.Maybe (Maybe(..))
+import Data.Tuple.Nested ((/\))
+import Effect (Effect)
+import KSF.HtmlRenderer.Models as HtmlRenderer
+import Lettera.Models (ArticleStub)
+import Mosaico.MostReadList as MostReadList
+
+data Hook
+  = AndraLaser (Array ArticleStub) (ArticleStub -> Effect Unit)
```

Comment:
I'd prefer this to be in English. I'll give a pass to what we're doing with categories since they're ever at most used in simple pattern matching but this use is more involved and I'd need to think about it where it is more and I'd prefer that part to stay monolingual.

Reply:
And how does one translate it? `ReadAlso` maybe?